src/row.rs

```diff
@@ -127,21 +136,13 @@ pub struct MappedRows<'stmt, F> {
     map: F,
 }
 
-impl<'stmt, T, F> MappedRows<'stmt, F>
```

Comment:
MappedRows::new isn't needed if we just implement `query_map` in terms of `query(...).mapped`. This is unrelated but accidentally got rolled into this patch. It doesn't change behavior or the public API at all.

Ditto for AndThenRows::new